### DISCUSSION TITLE
CefSharp.BrowserSubprocess.exe

### DISCUSSION THREAD

**Discussion by Mephisto090** [cefsharp/CefSharp#4455] [General]
Hej,

Error: ![image](https://user-images.githubusercontent.com/85995176/229922765-0b2b6135-cb02-4c38-a64a-a77d8b40ebec.png)

The file and similar files are no longer moved to the folder when compiling and lead to errors "CefSharp.BrowserSubprocess.exe"

CefSharp Version:
![image](https://user-images.githubusercontent.com/85995176/229923560-f1f7a13c-f71e-4206-b03a-baa1d24405b2.png)

Missing Files:
![image](https://user-images.githubusercontent.com/85995176/229925029-ed66b254-16d4-43cf-97a7-7c834cfcdf0d.png)
See marked files

regards

**Comment by Mephisto090** [comment]
Doesn't seem to be an error, where is the problem?

**Reply by amaitland** [reply]
https://github.com/cefsharp/CefSharp/wiki/Questions-&-Support#no-images-of-exceptions

What version of `.Net` are you targeting?

**Reply by Mephisto090** [reply]
> What version of `.Net` are you targeting?

.NET Framework 4.8

```
The ....\CefSharp.Common.111.2.20\buildTransitive\..\CefSharp\x86\CefSharp.BrowserSubprocess.Core.dll-Assembly was incorrectly specified as a file.
The ....\CefSharp.Common.111.2.20\buildTransitive\..\CefSharp\x86\CefSharp.BrowserSubprocess.exe-Assembly was incorrectly specified as a file.
For the referenced ...\CefSharp.Common.111.2.20\CefSharp\x86\CefSharp.Core.Runtime.dll-Assembly a different target processor is specified than for the application.
```

**Reply by amaitland** [reply]
https://codeblog.jonskeet.uk/2010/08/29/writing-the-perfect-question/#golden-rule

**Reply by Mephisto090** [reply]
> https://codeblog.jonskeet.uk/2010/08/29/writing-the-perfect-question/#golden-rule

The file and similar files are no longer moved to the folder when compiling and lead to errors "CefSharp.BrowserSubprocess.exe"
Why is this no longer moved to the project folder when compiling?
